include/engine.hpp

```diff
@@ -5,6 +5,8 @@
 
 #include "scenes.hpp"
 #include "textures.hpp"
+#include "leaderboard.hpp"
+#include <unistd.h>
```

Comment:
`<unistd.h> ` was used for including a `sleep` function for a smooth transition to a leaderboard chart. Since you ask to remove it, include would be removed as well.